### PR TITLE
New package: r-questionr

### PR DESCRIPTION
**Pull request by qwertos**

### CODE REVIEW

File: var/spack/repos/builtin/packages/r-questionr/package.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+
+class RQuestionr(RPackage):
+    """Set of functions to make the processing and analysis of surveys easier :
+    interactive shiny apps and addins for data recoding, contingency tables,
+    dataset metadata handling, and several convenience functions."""
+
+    homepage = "https://cloud.r-project.org/package=questionr"
+    url      = "https://cloud.r-project.org/src/contrib/questionr_0.7.4.tar.gz"
+
```
Comment (glennpj) [inline var/spack/repos/builtin/packages/r-questionr/package.py:17]: Need a `list_url`

**Review by glennpj — CHANGES_REQUESTED** [review]

**Comment by glennpj** [comment]
@alalazo I actually had already reviewed this PR, and requested changes. I should also note here that this is dependent on #25399.

**Comment by alalazo** [comment]
You're right @glennpj  I didn't notice the request was not addressed.

**Comment by qwertos** [comment]
list_url is there now. Forgot to hit resolve thread.

**Review by glennpj — DISMISSED** [review]

**Review by glennpj — APPROVED** [review]

**Pull request merged**